cmake/BLTPrivateMacros.cmake

```diff
@@ -129,52 +180,25 @@ macro(blt_setup_target)
         endif()
 
         if ( DEFINED BLT_${uppercase_dependency}_INCLUDES )
-            if ( BLT_${uppercase_dependency}_TREAT_INCLUDES_AS_SYSTEM )
-                target_include_directories( ${arg_NAME} SYSTEM PUBLIC
-                    ${BLT_${uppercase_dependency}_INCLUDES} )
-            else()
-                target_include_directories( ${arg_NAME} PUBLIC
-                    ${BLT_${uppercase_dependency}_INCLUDES} )
-            endif()
-        endif()
-
-        # Handle inherited information for object libraries in CMake versions
-        # less than 3.12, because target_link_libraries can't be called
```

Comment:
Whoa -- do we still need any of the pre-3.12 logic?

Reply:
It all went into the new inherit macro.  Now we just guard the target_link_library call against previous versions.  We might not even need the call in new versions but it does protect us against any inheritance we miss.